test/system/system.test.ts

```diff
@@ -430,6 +430,18 @@ test(
   },
 );
 
+test("inspect image that doesn't exist locally but pullable from docker hub", async (t) => {
+  const img = "alpine:3.10";
```

Comment:
Does this test assume I don't have alpine:3.10 in my local registry?

Reply:
If I understood you correctly, it runs `docker rmi` hoping that it would indeed remove it locally